docs/src/components/FeaturedInstrumentVideo.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+type FeaturedInstrumentVideoProps = {
+  category: string;
+  manufacturer: string;
+};
+
+export default function FeaturedInstrumentVideo({
+  category,
+  manufacturer,
+}: FeaturedInstrumentVideoProps) {
+  let VIDEO_URL =
+    "https://www.youtube.com/embed/xvFZjo5PgG0?si=oXQsWInrLBlhAoCB&mute=1";
+  let VIDEO_HERO = "NOT FOUND";
+
+  // console.warn('Instrument category: ', category);
+
+  switch (category) {
+    case "OSCILLOSCOPES":
+      VIDEO_URL =
+        "https://www.youtube.com/embed/NqPW7pG3NZs?si=P374jJmZNKatbExq";
+      VIDEO_HERO =
+        "Measure signal width and phase with a Tektronix oscilloscope";
+      break;
+
+    case "DAQ_BOARDS":
+      VIDEO_URL =
+        "https://www.youtube.com/embed/ph64DXjX1-8?si=MdY5XjyGhpbTEswx";
+      VIDEO_HERO = "Record temperature over time with a LabJack DAQ board";
+      break;
+
+    case "MULTIMETERS":
+      VIDEO_URL =
+        "https://www.youtube.com/embed/jaN89-ijxL0?si=ytXVo41QkjADx8us";
+      VIDEO_HERO = "Record voltage over time with an Agilent 34401A multimeter";
+      break;
+
+    case "POWER_SUPPLIES":
+      VIDEO_URL =
+        "https://www.youtube.com/embed/OEzThwlCU1s?si=9HcGT7Kij5h9GoSe";
+      VIDEO_HERO = "Measure a solar panel IV curve with a Keithley 2400";
+      break;
+
+    case "MOTOR_CONTROLLERS":
+      VIDEO_URL =
+        "https://www.youtube.com/embed/6NBlT6VWQW0?si=FqTfe1WJTMcoNilq";
+      VIDEO_HERO = "Send commands to a Polulu stepper motor driver";
+      break;
+  }
+
+  if (VIDEO_HERO == "NOT FOUND") {
+    return <></>;
+  }
```

Comment:
I think instead of a const object, we could put that in AirTable instead. Having another table that keeps tracks of all the data. We should not hardcode any data like this in the codebase. The airtable should be the single source of truth, and starlight is only for displaying them

Reply:
@itsjoeoui I have temporarily converted the switch statement to a constant object with a TODO comment as I failed to create table in Airtable initially.